src/main/webapp/groupsPage.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>The Solidarity Initiative</title>
+        <link rel="stylesheet" href="groupsPage.css">
+        <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+        <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
+        <script src="groupsPage.js"></script>
+    </head>
+
+    <body>
+        <div>
+            <h1 class="heading mdc-typography--headline3">Movements</h1>
+        </div>
+
+        <div class="card-gallery">
+            <div class="mdc-card">
+                <div class="mdc-card__media mdc-card__media--16-9" style="background-image: url(https://images.unsplash.com/photo-1591622414912-34f2a8f8172e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80);"></div>
+                <div class="title-container">
+                    <h2 class="text mdc-typography mdc-typography--headline6">Black Lives Matter</h2>
+                    <!-- <h3 class="demo-card__subtitle mdc-typography mdc-typography--subtitle2">by Kurt Wagner</h3> -->
+                </div>
+
+                <div class="desc-container mdc-typography mdc-typography--body2">
+                    <p class="text">
```

Comment:
Try to make this selector more specific. `text` is very generic, and you will likely add more text to this page that won't want the same styling.